### PR TITLE
ci: Switch to `/usr/bin/time` for measuring time + memory in physmon

### PR DESCRIPTION
**Pull request by paulgessinger**
I'm hoping this circumvents the problem of missing memory and time measurements. They're possibly less accurate, but less buggy for now. Should produce the same CSV as before, and I still use the custom python tool to convert them into a combined CSV that's published on the tracker.

### CODE REVIEW

File: Tests/UnitTests/Plugins/DD4hep/CMakeLists.txt [hunk 1]
```diff
@@ -41,5 +42,5 @@ if (EXISTS ${DD4hep_DIR}/${CMAKE_INSTALL_LIBDIR})
     endif()
   endforeach()
 else()
-    message(WARNING "DD4hep libraries not found under ${DD4hep_DIR}/${CMAKE_INSTALL_LIBDIR}, DD4hep tests will not be built.")
-endif()
\ No newline at end of file
+    message(WARNING "DD4hep libraries not found under, DD4hep tests will not be built.")
```
Comment (AJPfleger) [inline Tests/UnitTests/Plugins/DD4hep/CMakeLists.txt:45]:
```suggestion
    message(WARNING "DD4hep libraries not found, DD4hep tests will not be built.")
```

Comment (paulgessinger) [reply Tests/UnitTests/Plugins/DD4hep/CMakeLists.txt:45]: Actually I wanted to remove this change from this PR, sorry!

File: CI/physmon/phys_perf_mon.sh [hunk 2]
```diff
@@ -1,7 +1,19 @@
...
 
 set -e
-set -x
+
+# helper function to selectively print and run commands without a subshell
+function run() {
+    set -x
+    "$@"
+    { set +x;   } 2> /dev/null
+
```
Comment (andiwand) [inline CI/physmon/phys_perf_mon.sh:10]:
```suggestion
```

**Review by andiwand — APPROVED** [review]

**Review by andiwand — APPROVED** [review]

**Comment by acts-project-service** [comment]
## :red_circle: Athena integration test results [d455e02431f50c4127640adae18996a37a191630]
### :red_circle: Some tests have failed!
Please investigate the [pipeline](https://gitlab.cern.ch/acts/acts-athena-ci/-/pipelines/7016919)!

|status|job|report|
|------|---|------|
|:green_circle: | [run_unit_tests](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806418)||
|:green_circle: | [test_ActsEFTrackFit](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806417)||
|:green_circle: | [test_ActsBenchmarkWithSpot](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806416)||
|:green_circle: | [test_ActsConversionWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806415)||
|:red_circle: | [test_ActsWorkflow](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806414)||
|:green_circle: | [test_ActsValidateAmbiguityResolution](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806413)||
|:green_circle: | [test_ActsValidateResolvedTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806412)||
|:green_circle: | [test_ActsValidateTracks](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806411)||
|:green_circle: | [test_ActsValidateActsCoreSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806410)||
|:green_circle: | [test_ActsValidateActsSpacePoints](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806409)||
|:green_circle: | [test_ActsValidateSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806408)||
|:green_circle: | [test_ActsValidateOrthogonalSeeds](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806407)||
|:green_circle: | [test_ActsValidateClusters](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806406)||
|:green_circle: | [test_ActsPersistifyEDM](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806405)||
|:green_circle: | [test_ActsGSFRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806404)||
|:green_circle: | [test_ActsKfRefitting](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806403)||
|:red_circle: | [test_ActsExtrapolationAlgTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806402)||
|:green_circle: | [test_ActsITkTest](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806401)||
|:green_circle: | [run_workflow_tests_run4_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806400)||
|:red_circle: | [run_workflow_tests_run2_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806399)||
|:green_circle: | [run_workflow_tests_run2_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806398)||
|:green_circle: | [run_workflow_tests_run3_mc](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806397)||
|:green_circle: | [run_workflow_tests_run3_data](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806396)||
|:red_circle: | [run_art_test: test_data18_13TeV_1000evt](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806395)||
|:green_circle: | [run_art_test: test_ttbarPU40_reco](https://gitlab.cern.ch/acts/acts-athena-ci/-/jobs/36806394)||

**Pull request merged**